online-resources/tech-tests-and-interviews.md

```diff
@@ -12,10 +12,16 @@
 * [JS specific - The Definitive JavaScript Handbook for your next developer interview](https://medium.freecodecamp.org/the-definitive-javascript-handbook-for-a-developer-interview-44ffc6aeb54e)
 * [Learn about Scalability to use in System Design Interview - Harvard summer school lecture](https://www.youtube.com/watch?v=-W9F__D3oY4)
 
+### Practice
```

Comment:
Excellent idea, some duplication of Codewars & freeCodeCamp with online resources, but think it's valid as folk may not know they're great tech test resource